### ISSUE TITLE
how to solve training data and train prototxt

### ISSUE THREAD

**Issue by lucyandjason** [shizenglin/Deep-NCL#10]
Hello, I would like to ask the following questions: 
1. The density map used for training is. Mat or. JPG converted.What format is the density. m you gave us?
2. Is the prototxt used for training network_vgg_v1.prototxt?What does that have to do with what create_prototxt.py generates which looks like it's only going to be the loss layer?Is it used to modify network_vgg_v1.prototxt?
3. How do you divide the number of trains and tests?Shanghaitech data set I use.
I take the liberty to ask a lot of questions. Please forgive my unprofessionalism. Looking forward to your reply.
